client/src/components/admin/ModalAddEditNews.vue

```diff
@@ -21,11 +21,14 @@
         </div>
         <div class="modal-body">
           <!-- TODO: Fix modification of props -->
+          <!-- 🤨 ⬆️ -->
           <BaseInput v-model="newsObject.title" label="Titel" />
           <BaseTextarea v-model="newsObject.message" label="Nachricht" />
-          <!-- <BaseInput v-model="newsObject.from" label="Gültig ab" />
-          <BaseInput v-model="newsObject.till" label="Gültig bis" />-->
+          <!-- TODO: Icon select. For now just paste the name of an icon as this component needs refactoring anyway 
+          Later a picker would be nice -->
+          <BaseInput v-model="newsObject.icon" label="Icon" />
```

Comment:
Es sollte ein Default Icon gesetzt werden. (z.B. bi bi-info-circle).
Dann erkennt man auch direkt in welchem Format, man hier etwas einfügen soll. (Evtl. link to Bootstrap Icons?)

Reply:
Hab ja extra nen Kommentar dazu geschrieben. Warum sol ich da jetzt was schickes einbauen, wenn die Komponente eh kaputt ist und umgebaut wird;)

Reply:
Dann müsste aber für den Icon BaseInput aber wenigstens das Prop "is-required" auf false gesetzt werden. Sonst wird der Rahmen rot.